dask/dataframe/tests/test_groupby.py

```diff
@@ -1313,6 +1313,9 @@ def test_fillna(axis, group_keys, method, limit):
     )
     ddf = dd.from_pandas(df, npartitions=2)
 
+    def f(obj, *args, **kwargs):
```

Comment:
I feel like this function makes the test more complicated to read than necessary. It's DRY but not DAMP. 

Can we extract the cases where `f` is used to call `ffill` and `bfill` into dedicated tests `test_{b|f}fill` analogous to `test_fillna`?

Reply:
Good point. I updated the tests.